test/restapitest/uninstall.go

```diff
@@ -72,50 +87,64 @@ func newUninstallApp(th *Helper) *goapp.App {
 }
 
 func testUninstall(th *Helper) {
-	creq := apps.CallRequest{
+	info := apps.CallRequest{
 		Call: *apps.NewCall(builtin.PathDebugKVInfo).WithExpand(apps.Expand{ActingUser: apps.ExpandSummary}),
 	}
+	pollute := func(n int) {
+		th.Run(fmt.Sprintf("add %v garbage KV entries", n), func(th *Helper) {
+			cresp := th.HappyAdminCall(builtin.AppID, apps.CallRequest{
+				Call:   *apps.NewCall(builtin.PathDebugStorePollute).WithExpand(apps.Expand{ActingUser: apps.ExpandSummary}),
+				Values: map[string]interface{}{"count": fmt.Sprintf("%v", n)},
+			})
+			require.Equal(th, apps.CallResponseTypeOK, cresp.Type)
+		})
+	}
 
+	pollute(599)
```

Comment:
Why 599 and 1401?

Reply:
The "window" size for paging is 1000, so I wanted to create 2K, but unevenly, in case the creation order matters for `ListKeys`. Really, Maybe should be 1401 and then 599.